project/profiles/capacity4more/modules/c4m/og/c4m_og/c4m_og.module

```diff
@@ -945,16 +945,21 @@ function c4m_og_get_user_group_nodes_by_bundle($bundle = NULL, $account = NULL)
     return $groups;
   }
 
-  $conditions = array(
-    'status' => NODE_PUBLISHED,
```

Comment:
Yup, but it's needed since it applies the filtering (line 950) only when a filter is on.